Release all integrations for 7.74 (#21980)
DataDog/integrations-core: sqlserver/CHANGELOG.md

<!-- towncrier release notes start -->

## 22.12.0 / 2025-11-26

***Added***:

> * Migrate SQL Server to new schema collector. This collector provides improved performance in the agent and allows the backend to handle larger schema collections. ([#21734](https://github.com/DataDog/integrations-core/pull/21734))

Reviewer: ```suggestion
* Migrate SQL Server to new schema collector. This collector provides improved performance in the Agent and allows the backend to handle larger schema collections. ([#21734](https://github.com/DataDog/integrations-core/pull/21734))
```

> * Add DBM agent health events to SQLServer. Events include basic initialization check, unhandled error, and missed collection. ([#21868](https://github.com/DataDog/integrations-core/pull/21868))

Reviewer: ```suggestion
* Add DBM Agent health events to SQLServer. Events include basic initialization checks, unhandled errors, and missed collections. ([#21868](https://github.com/DataDog/integrations-core/pull/21868))
```

* Upgrade base version for Postgres, MySQL, SQLServer ([#21906](https://github.com/DataDog/integrations-core/pull/21906))

***Fixed***:

> * Support `only_custom_queries` configuration option in sqlserver integration. ([#21304](https://github.com/DataDog/integrations-core/pull/21304))

Reviewer: ```suggestion
* Support `only_custom_queries` configuration option in the SQL Server integration ([#21304](https://github.com/DataDog/integrations-core/pull/21304))
```
